### PR TITLE
Netcdf library include options for compilation

### PR DESCRIPTION
**Pull request by jdmattern-noaa**
## Additions

-

## Removals

-

## Changes

-

## Testing

1.

## Notes

-

## Todos

-

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows project standards (link if applicable)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Target Environment support

- [ ] Windows
- [ ] Linux
- [ ] Browser

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: src/fortran_routing/Reservoir_Binding/Reservoirs/makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-NETCDFINC := $(shell nc-config --fflags)
+NETCDFINC := -I$(NETCDFINC) $(shell nc-config --fflags)
```
Comment (jdmattern-noaa) [inline src/fortran_routing/Reservoir_Binding/Reservoirs/makefile:1]: @hellkite500 @jameshalgren Would the be the right update to make to the makefile permanently?

File: src/python_routing_v02/compiler.sh [hunk 2]
```diff
@@ -19,6 +19,7 @@ fi
 
 #if you have custom static library paths, uncomment below and export them
 #export LIBRARY_PATH=<paths>:$LIBRARY_PATH
+#export NETCDFINC=/usr/include/openmpi-x86_64/
```
[the remark this answers is not in the document]
Comment (jdmattern-noaa) [reply src/python_routing_v02/compiler.sh:22]: @jhrehanoaa You might be aware of this, but you will want to to do this for nwm_routing:

cd ../nwm_routing

pip install -e .

File: src/fortran_routing/Reservoir_Binding/Reservoirs/makefile (hunk 1 above)
Comment (hellkite500) [reply src/fortran_routing/Reservoir_Binding/Reservoirs/makefile:1]: Yeah, it just prepends the ENV variable `NETCDFINC` if it exists.

**Comment by awlostowski-noaa** [comment]
We rolled these changes into recently merged #327. Thanks!

**Pull request closed without merging**